.pre-commit-config.yaml

```
@@ -0,0 +1,7 @@
+repos:
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 0.11.1
+    hooks:
+      - id: nbqa-black
+        args: [--nbqa-mutate]
+      - id: nbqa-flake8
```

Might be a good idea to pin `black` and `flake8` here, to avoid surprises / mismatches between local and CI environments?
```suggestion
      - id: nbqa-black
        additional_dependencies: [black==21.5b2]
        args: [--nbqa-mutate]
      - id: nbqa-flake8
        additional_dependencies: [flake8==3.9.2]
```